api/model.go

```diff
@@ -932,6 +932,7 @@ func toVolumeResource(v *longhorn.Volume, ves []*longhorn.Engine, vrs []*longhor
 			actions["pvcCreate"] = struct{}{}
 			actions["updateDataLocality"] = struct{}{}
 			actions["updateAccessMode"] = struct{}{}
+			actions["snapshotList"] = struct{}{}
```

Comment:
Longhorn cannot list snapshots for a detached volume...

Reply:
Arrgh..this is partial fix of the 5 skipped tests... let me remove it.